tests/keras/backend/backend_test.py

```diff
@@ -1159,7 +1159,7 @@ def test_in_top_k(self):
         for k in range(num_classes + 1):
```

Comment:
Thank you for the PR, @abhaikollara. How about changing here as `range(1, num_classes + 1)`?

Reply:
Done. Should the test be skipped though ? If this is merged all consequent builds will be marked as failing.

Reply:
Do you have plans to investigate the weird behavior of `in_top_k` in CNTK?